test/e2e_node/cpu_manager_test.go

```diff
@@ -47,27 +49,130 @@ type ctnAttribute struct {
 	ctnName    string
 	cpuRequest string
 	cpuLimit   string
+	memRequest string
+	memLimit   string
+	command    string
+}
+
+func (ca ctnAttribute) Clone() ctnAttribute {
+	return ctnAttribute{
+		ctnName:    ca.ctnName,
+		cpuRequest: ca.cpuRequest,
+		cpuLimit:   ca.cpuLimit,
+		memRequest: ca.memRequest,
+		memLimit:   ca.memLimit,
+		command:    ca.command,
+	}
+}
+
+func (ca ctnAttribute) WithDefaults() ctnAttribute {
+	cmd := "grep Cpus_allowed_list /proc/self/status | cut -f2 && sleep 1d"
```

Comment:
I wonder if the allowed cpus will still be reflected properly under the status if cgroup is of v2, If not mistaken I saw that empty, but currently I don't have a cluster with v2. I'll deploy one and check.